pkgs/games/gscrabble/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, buildPythonApplication, fetchFromGitHub
+, gtk3, gobjectIntrospection, wrapGAppsHook, gst_all_1
+, python3, python3Packages, gnome3, hicolor-icon-theme }:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "gscrabble";
+  version = "0.1.2";
+
+  src = fetchFromGitHub {
+    owner = "RaaH";
+    repo = "gscrabble";
+    rev = "4609e6be4867836afa98dd1d2c4f5296d9706416";
+    sha256 = "0j6102ylzwlqc2bvswff49m8psbi3qikv63m95ps8sr8yhwi2idp";
+  };
+
+  doCheck = false;  # no tests available
+
+  buildInputs = with gst_all_1; [
+    gst-plugins-base gst-plugins-good gst-plugins-ugly gst-plugins-bad
+  ];
+
+  nativeBuildInputs = [ 
+    python3
+    gobjectIntrospection # populate GI_TYPELIB_PATH
+    wrapGAppsHook
+   ];
+
+  propagatedBuildInputs = with python3Packages; [
+    gtk3
+    gst-python
+    pygobject3
+  ];
+
+  propagatedUserEnvPkgs = [
+    gnome3.gnome-icon-theme-symbolic
+    hicolor-icon-theme
+  ];
+
+  makeWrapperArgs = "--prefix PYTHONPATH : $out/share/GScrabble/modules";
+
+  meta = with stdenv.lib; {
+    description = "A crossword puzzle game";
+    homepage = https://github.com/RaaH/gscrabble/;
+    license = licenses.gpl2;
+    platforms = platforms.linux;
+    maintainers = [ maintainers.ciil ];
```

Comment:
Who is @ciil?

Reply:
As this is another LGame, I'm guessing @bignaux used one of ltris or lbreakout2, which I maintain, as a template ;).

Reply:
Anyway, are you willing to maintain this?

Reply:
No, I'd like to ask @bignaux to please remove me from the maintainers list.

Reply:
yes it's a wip i had already fix that in my tree since yesterday. The question was about to restore the gnome-icon-theme-symbolic , and yes it's a copy paste from the gshogi. I should admit a lot of suggestion here, and i've to split the theme in another bug to have answer about revert of this. Thanks for review.

Reply:
I think it is most likely that the README just has not been updated and Adwaita will be good enough. Try opening an issue asking about it in the gscrobble repo.